src/Parsers/CSVParser.php

```diff
@@ -18,6 +18,7 @@ class CSVParser implements StreamParserInterface
 	protected $reader, $source, $headers, $currentLine;
 
 	public static $delimiters = [",", ";"];
+	public static $delimiter = ",";
```

Comment:
I'm thinking of changing '$delimiter' by '$fieldDelimiter' so it's more clear. What do you think?